### PR TITLE
[3.x] Add Server types

### PR DESCRIPTION
**Pull request by tfevens**
This PR adds the newly announced `types` to the Server class.

Available types were added to the ServerTypes helper class, as implemented in f3c91f348ea8a8e802cc62b7fe8332f4dc792b6a.

### CODE REVIEW

File: src/Resources/Server.php [hunk 1]
```diff
@@ -18,6 +18,13 @@ class Server extends Resource
      */
     public $name;
 
+    /**
+     * The name of the server.
```
Comment (jbrooksuk) [inline src/Resources/Server.php:22]: This should be:

```suggestion
     * The type of the server.
```

File: src/Resources/ServerTypes.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+namespace Laravel\Forge\Resources;
+
+class ServerTypes
+{
+    const APP = 'app';
+    const WEB = 'web';
+    const LOADBALANCER = 'loadbalancer';
+    const CACHE = 'cache';
+    const DATABASE = 'database';
```
Comment (jbrooksuk) [inline src/Resources/ServerTypes.php:11]: We're missing the `worker` type here.

**Review by jbrooksuk — CHANGES_REQUESTED** [review]

**Comment by tfevens** [comment]
Thanks for catching!  Updated the PR with the requested fixes.

**Comment by jbrooksuk** [comment]
LGTM!

**Comment by driesvints** [comment]
@tfevens the PR contains some of the commits from master. Please re-send this with you your own commits and no merge commits. Thanks

**Pull request closed without merging**